p2p/peerexchange/bookchecker_test.go

```diff
@@ -223,7 +223,11 @@ func setupOverNodes(t *testing.T, nodesCount int) ([]host.Host, []*Discovery) {
 	for i := 0; i < nodesCount; i++ {
 		cm, err := connmgr.NewConnManager(40, 100, connmgr.WithGracePeriod(30*time.Second))
 		require.NoError(t, err)
-		h, err := libp2p.New(libp2p.ConnectionManager(cm))
+		lopts := []libp2p.Option{
+			libp2p.ConnectionManager(cm),
+			libp2p.ListenAddrStrings("/ip4/0.0.0.0/tcp/0"),
```

Comment:
Is port 0 correct? The default for spacemesh is `"/ip4/0.0.0.0/tcp/7513"`

Reply:
Port 0 is special - it will randomly pick up a free port.